src/DataLib/fendl32B_retrofit/tendl_processing.py

```diff
@@ -49,14 +49,15 @@ def search_for_files(dir = '.'):
     dir = Path(dir)
 
     file_info = {}
-    for file in (p for p in dir.glob('*') if p.suffix in {'.tendl', '.endf'}):
-        if file.is_file() and file.with_suffix('.pendf').is_file():
-            element, A = get_isotope(file.stem)
-            file_info[f'{element}{A}'] = {
-                'Element'       :                              element,
-                'Mass Number'   :                                    A,
-                'File Paths'    :   (file, file.with_suffix('.pendf'))
-            }
+    for suffix in ['tendl', 'endf']:
+            for file in dir.glob(f'*.{suffix}'):
+                if file.with_suffix('.pendf').is_file():
+                    element, A = get_isotope(file.stem)
+                    file_info[f'{element}{A}'] = {
+                        'Element'       :                             element,
+                        'Mass Number'   :                                   A,
+                        'File Paths'    :  (file, file.with_suffix('.pendf'))
+                    }
```

Comment:
This is indented too much?
```suggestion
        for file in dir.glob(f'*.{suffix}'):
            if file.with_suffix('.pendf').is_file():
                element, A = get_isotope(file.stem)
                file_info[f'{element}{A}'] = {
                    'Element'       :                             element,
                    'Mass Number'   :                                   A,
                    'File Paths'    :  (file, file.with_suffix('.pendf'))
                }
```

Reply:
I think you may be looking at an outdated version. I moved that indentation back a few commits ago.